vlm/src/main/scala/geotrellis/contrib/vlm/gdal/package.scala

```diff
@@ -0,0 +1,69 @@
+package geotrellis.contrib.vlm
+
+import geotrellis.gdal._
+import geotrellis.raster._
+import geotrellis.proj4.CRS
+import geotrellis.raster.reproject.Reproject.{Options => ReprojectOptions}
+import geotrellis.vector.Extent
+
+import cats.syntax.option._
+
+package object gdal {
+
+  implicit class GDALRasterExtentMethods(val self: RasterExtent) {
+
+    /**
+      * This method copies gdalwarp -tap logic:
+      *
+      * The actual code reference: https://github.com/OSGeo/gdal/blob/v2.3.2/gdal/apps/gdal_rasterize_lib.cpp#L402-L461
+      * The actual part with the -tap logic: https://github.com/OSGeo/gdal/blob/v2.3.2/gdal/apps/gdal_rasterize_lib.cpp#L455-L461
+      *
+      * The initial PR that introduced that feature in GDAL 1.8.0: https://trac.osgeo.org/gdal/attachment/ticket/3772/gdal_tap.patch
+      * A discussion thread related to it: https://lists.osgeo.org/pipermail/gdal-dev/2010-October/thread.html#26209
+      *
+      */
+    def alignTargetPixels: RasterExtent = {
```

Comment:
Worrying about this logic here; should it be there at all? However without it we won't get tests won't pass :/